config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -114,20 +129,20 @@ periodics:
     containers:
     - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20210917-ee1e7c845b-1.21
       command:
-      - runner.sh
-      - bash
+        - runner.sh
       args:
-      - -c
-      - |
-        result=0
-        ./scripts/ci-test-coverage.sh || result=$?
-        cp coverage.* ${ARTIFACTS}
-        cd ../../k8s.io/test-infra/gopherage
-        GO111MODULE=on go build .
-        ./gopherage filter --exclude-path="zz_generated,generated\.go" "${ARTIFACTS}/coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
-        ./gopherage html "${ARTIFACTS}/filtered.cov" > "${ARTIFACTS}/coverage.html" || result=$?
-        ./gopherage junit --threshold 0 "${ARTIFACTS}/filtered.cov" > "${ARTIFACTS}/junit_coverage.xml" || result=$?
-        exit $result
+        - bash
```

Comment:
just to confirm, this is just formatting change and moving `bash` to args?

Reply:
Correct. Sorry for the extraneous diffs, I should have reverted those but they should be no-ops.